### PR TITLE
Patch/reorg 12 feb17

### PR DESCRIPTION
**Pull request by johnmay**
First batch of reorganisations and spring cleaning. 

**API Changes**
There are some API changes here in terms of where classes are located (in which module) which I feel are acceptable as including ```cdk-bundle``` will still allow compilation. 

I did move and encapsulate the ``MathTools`` class as this was only used by a specific class and should not have been public in the first place. Again I feel this change is acceptable, as a rule of thumb I would say "is it chemistry related? Should or could someone expect to be calling this method". Ideally I would encapsulate the EStateFragments class as well but I feel this could be used externally.

**Deprecation**
I've added some more deprecation warnings to old classes.

### CODE REVIEW

File: legacy/src/main/java/org/openscience/cdk/normalize/CDKUtilities.java [hunk 1]
```diff
@@ -2,23 +2,24 @@
...
- * GNU Lesser General Public License for more details.
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public
+ * License for more details.
  *
  * You should have received a copy of the GNU Lesser General Public License
  * along with this program; if not, write to the Free Software
- * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 U
```
Comment (johnmay) [inline legacy/src/main/java/org/openscience/cdk/normalize/CDKUtilities.java]: Odd not sure where there came from.

File: legacy/src/test/java/org/openscience/cdk/smiles/DeduceBondSystemToolTest.java [hunk 2]
```diff
@@ -29,13 +33,13 @@
...
 import org.openscience.cdk.interfaces.IAtomType.Hybridization;
-import org.openscience.cdk.interfaces.IBond.Order;
 import org.openscience.cdk.interfaces.IBond;
+import org.openscience.cdk.interfaces.IBond.Order;
 import org.openscience.cdk.ringsearch.AllRingsFinder;
-import org.openscience.cdk.silent.SilentChemObjectBuilder;
 import org.openscience.cdk.silent.Atom;
 import org.openscience.cdk.silent.AtomContainer;
 import org.openscience.cdk.silent.Bond;
+import org.openscience.cdk.silent.SilentChemObjectBuilder;
```
Comment (johnmay) [inline legacy/src/test/java/org/openscience/cdk/smiles/DeduceBondSystemToolTest.java]: I actually disagree with that design in general, but regardless in this case it is
  a) in a test class (i.e. it is already decoupled)
  b) deprecated

Comment (egonw) [inline legacy/src/test/java/org/openscience/cdk/smiles/DeduceBondSystemToolTest.java]: (yes, tests actually are fine... actually, I find it hard to see in which file changes are in the web gui... overlooked that; but this was not a change request in the first place...)

**Comment by johnmay** [comment]
Fixed

**Pull request merged**